scripts/EN/Harry

```diff
@@ -2172,7 +2172,7 @@ Area(0x25); // 0x277a: 00001026
 PlaySoundFx(0xd, 0x1); // 0x2780: 00002072
 PlayMusic(0x66, 0x1); // 0x2788: 0000206c
 LoadSecondEffect(0x54, 0x1, 0x1, 0x1); // 0x2790: 0000406e
-Choice((60)"I...", (61)"A ) Instantly pushed the monster away with my hands.", (62)"B ) Readied the handgun and fired."); // 0x279c: 00003011
+Choice((60)"I…", (61)"A) instantly pushed the monster away with my hands.", (62)"B) readied the handgun and fired."); // 0x279c: 00003011
```

Comment:
Removed the capitalization from instantly so it would feel more like this was completing the sentence beginning with "I..."